indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -365,6 +365,7 @@ public TaskStatus call()
                                   )
                               );
 
+                              command.add(String.format("-Ddruid.indexer.taskId=%s", task.getId()));
```

Comment:
This should be fetched from `Task`, see https://github.com/druid-io/druid/pull/2955